src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,92 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_PROPERTY_SIZE = 10;
+    private static final Color DEFAULT_COLOR = Color.WHITE;
+    private static final int NUMBER_OF_FIGURES = 5;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(NUMBER_OF_FIGURES)) {
+            default:
+                return getCircle(FigureType.RANDOM);
+            case 0:
+                return getRectangle(FigureType.RANDOM);
+            case 1:
+                return getRightTriangle(FigureType.RANDOM);
+            case 2:
+                return getSquare(FigureType.RANDOM);
+            case 3:
+                return getIsoscelesTrapezoid(FigureType.RANDOM);
+        }
+    }
+
+    public Figure getDefaultFigure() {
+        return getCircle(FigureType.DEFAULT);
+    }
+
+    private int getRandomProperySize() {
+        return random.nextInt(DEFAULT_PROPERTY_SIZE);
+    }
+
+    private Figure getCircle(FigureType figureType) {
+        if (figureType == FigureType.DEFAULT) {
+            return new Circle(colorSupplier.getRandomColor(),
+                    getRandomProperySize());
+        } else {
+            return new Circle(DEFAULT_COLOR,
+                    DEFAULT_PROPERTY_SIZE);
+        }
+    }
+
+    private Figure getSquare(FigureType figureType) {
+        if (figureType == FigureType.DEFAULT) {
+            return new Square(colorSupplier.getRandomColor(),
+                    getRandomProperySize());
+        } else {
+            return new Square(DEFAULT_COLOR,
+                    DEFAULT_PROPERTY_SIZE);
+        }
+    }
```

Comment:
all of those `if`s along with this `FigureType` are too much effort just to implement a default figure with hardcoded values, remove the described above from all of the similar methods and make `getDefaultFigure()` just return 
`new Circle(DEFAULT_COLOR, DEFAULT_PROPERTY_SIZE)`